inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -163,7 +163,8 @@ ndots: 2
 # Can be coredns, coredns_dual, manual or none
 dns_mode: coredns
 # Set manual server if using a custom cluster DNS server
-# manual_dns_server: 10.x.x.x
+# manual_dns_server:
+# - 10.x.x.x
```

Comment:
Should we enable to specify multiple servers on this option?

If so, the name should be `manual_dns_servers` instead.
The above comment of line 165 also needs to be updated.